docs/sources/docker-hub-enterprise/adminguide.md

```diff
@@ -1,104 +0,0 @@
-no_version_dropdown: true
```

Comment:
This was discussed last week. Given the magnitude of the change, the decision was made to allow breaks for this one time.  If I have time, I will try to build redirects but given the amount of testing and other projects I have online -- it is a nice-to-have.

The DHE material was being copied over from a private repository.  New release processes make this copying no longer necessary.